.buildkite/ftr_configs.yml

```diff
@@ -461,7 +457,15 @@ enabled:
   - x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/rule_creation/configs/ess.config.ts
   - x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/actions/configs/serverless.config.ts
   - x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/actions/configs/ess.config.ts
- 
+  - x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/prebuilt_rules/configs/serverless.config.ts
```

Comment:
Why can't we just have `prebuilt_rules` group?

Reply:
Because all of these folders have different configs which are specific to the tests. For example, for `large_prebuilt_rules_package`, we need to configure the env to use a max of 800mb to tests possible OOM issues. Similarly with the others; the specific configs are passed to the ess.config.ts and serveres.config.ts files